t/t1007-flux-account-users.t

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+test_description='Test flux-account commands'
```

Comment:
user commands?

Reply:
Yeah, all of these test descriptions should be updated/made clearer. Thanks for pointing that out!